integration-tests/api/__tests__/store/products.js

```diff
@@ -471,8 +471,18 @@ describe("/store/products", () => {
           console.log(err)
         })
 
-      expect(response.data.products).toHaveLength(5)
-      expect(response.data.products).toEqual(
+      const products = response.data.products
+
+      expect(products).toHaveLength(5)
+
+      const testProduct = products.find((p) => p.id === testProductId)
+      expect(testProduct.variants).toHaveLength(3)
+
+      for (const variant of testProduct.variants) {
```

Comment:
self-review: We added a check on the number of expected relations in order to be able to add `arrayContaining` below since the order matters and it can be different at the moment it is fetched